### PR TITLE
Update angular second attempt

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/package.json [hunk 1]
```diff
@@ -31,95 +32,94 @@
...
-    "webpack-bundle-analyzer": "3.1.0"
+    "source-map-explorer": "2.0.0",
+    "ts-node": "8.2.0",
+    "tslint": "5.17.0",
+    "tslint-angular": "3.0.2",
+    "typescript": "3.4.5",
+    "webpack-bundle-analyzer": "3.3.2"
   },
   "repository": {
     "type": "git",
```
Comment (99999989) [inline alv-portal-ui/package.json]: you have to update the `engines` property, too.
`"node": ">= 10.9.0",`

File: alv-portal-ui/src/app/candidate/candidate-search/candidate-detail/candidate-detail.component.html [hunk 2]
```diff
@@ -30,13 +30,13 @@
                ngbDropdownMenu>
 
             <button (click)="prev()"
-                    [disabled]="!(prevEnabled$ | async)"
+                    [disabled]="(prevEnabled$ | async) === false"
```
Comment (99999989) [inline alv-portal-ui/src/app/candidate/candidate-search/candidate-detail/candidate-detail.component.html]: I prefer to have spaces around the `===` like here. But either always or never.

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts [hunk 3]
```diff
@@ -126,10 +126,12 @@ export class EmploymentComponent extends AbstractSubscriber implements OnInit {
 
     this.parentForm.addControl(JobPublicationFormValueKeys.EMPLOYMENT, this.employment);
     this.setupWorkload();
+  }
+
+  public ngAfterViewInit() {
```
Comment (99999989) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts]: Is this change really needed?

File: alv-portal-ui/src/scss/overrides/em.scss [hunk 4]
```diff
@@ -0,0 +1,6 @@
+@import "../common/colors";
+
+em {
+  padding: 0 0.125rem;
+  background-color: $color-light-yellow;
+}
```
Comment (99999989) [inline alv-portal-ui/src/scss/overrides/em.scss]: I would add the following CSS rule: 
```
markdown em {
  padding: 0;
  background-color: $color-white;
}
```

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts (hunk 3 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts]: hmmm, seems like no

File: alv-portal-ui/package.json (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/package.json]: thanks!

File: alv-portal-ui/src/app/candidate/candidate-search/candidate-detail/candidate-detail.component.html [hunk 5]
```diff
@@ -30,13 +30,13 @@
                ngbDropdownMenu>
 
             <button (click)="prev()"
-                    [disabled]="!(prevEnabled$ | async)"
+                    [disabled]="(prevEnabled$ | async)===false"
```
Comment (birom) [inline alv-portal-ui/src/app/candidate/candidate-search/candidate-detail/candidate-detail.component.html]: I'm just curious. Causes the old sysntax linting error?

Comment (ganqqwerty) [reply alv-portal-ui/src/app/candidate/candidate-search/candidate-detail/candidate-detail.component.html]: yes, now it's an error, not warning

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts [hunk 6]
```diff
@@ -98,7 +95,7 @@ export class EmploymentComponent extends AbstractSubscriber implements OnInit {
   }
 
   ngOnInit(): void {
-    const { workloadPercentageMin, workloadPercentageMax, duration, immediately, startDate, endDate, workForms } = this.employmentFormValue;
+    const {workloadPercentageMin, workloadPercentageMax, duration, immediately, startDate, endDate, workForms} = this.employmentFormValue;
```
Comment (birom) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/employment/employment.component.ts]: Did we change the tslint rules? A space was earlier mandatory  between the ``{`` and the ``workloadPercentageMin``.

File: alv-portal-ui/package.json [hunk 7]
```diff
@@ -3,9 +3,10 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
+    "ng-update": "ng update @angular/cli @angular/core --allow-dirty",
     "start": "npm run start:staging",
     "start:dev": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.dev.json",
-    "start:staging": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.staging.json",
+    "start:staging": "npm run i18n && ng serve --base-href / --watch --proxy-config proxy.conf.staging.json --host 0.0.0.0",
     "start:mobile": "npm run i18n && ng serve --host 0.0.0.0 --base-href / --watch --proxy-config proxy.conf.staging.json",
```
Comment (99999989) [inline alv-portal-ui/package.json]: Maybe you can remove the `start:mobile` profile now as it's just a duplicate now.
